django_freeradius/base/models.py

```diff
@@ -796,6 +777,29 @@ def prefix_add(self, prefix, n, password_length=BATCH_DEFAULT_PASSWORD_LENGTH):
         self.full_clean()
         self.save()
 
+    def get_or_create_user(self, row, users_list, generated_passwords, password_length):
```

Comment:
seprated this logic from add function  to get or create user conditionally. This will help extended app like openwisp_radius to skip creating user in case of email exists and email field is unique.

Reply:
:+1: